index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="initial-scale=1, width=device-width" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap" />
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" />
+  </head>
+  <body>
+    <header class="header">
+      <img class="logo" src="./logo.png" alt="logo">
```

Comment:
- [ ] It's part of a convention that the logo should navigate to the home page when clicked. So, I suggest the logo should be inside an anchor tag `<a>`.  This will improve semantics, and user experience(UX).